packages/ui/src/components/atoms/Select/Select.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs'
```

Comment:
Maybe I got it wrong but aren't we planning to remove the storybook from `faststore/ui`?